manual/2-populate-the-structure/templates/quickstart/quickstart-template.md

```diff
@@ -0,0 +1,191 @@
+| Section                       | Format                              | Required  | ID                               |
+|:------------------------------|:------------------------------------|:----------|:---------------------------------|
+| Title                         | H1                                  | Yes       | `QST-TITLE`                      |
+| Subtitle                      | Single bold sentence                | Yes       | `QST-SUBTITLE`                   |
+| Access callout                | Note-type callout                   | No        | `QST-ACCESS`                     |
+| Callouts                      | Tip, Note, Important, Caution       | No        | `QST-CALLOUTS`                   |
+| Overview                      | Paragraph                           | Yes       | `QST-OVERVIEW`                   |
+| "Before you start" title      | H2                                  | No        | `QST-STRUCT-BEFORE-START-H2-TEXT`|
+| "Before you start" list       | Bullet list                         | No        | `QST-BEFORE-START`               |
+| Task title                    | H2                                  | Yes       | `QST-TASK-TITLE`                 |
+| Task intro                    | Paragraph                           | No        | `QST-TASK-INTRO`                 |
+| Task callout                  | Callout                             | No        | `QST-TASK-CALLOUTS`              |
```

Comment:
You use the lable `QTS-TASK-CALLOUTS` (plural), so update the other columns to the plural form as well (Task callout -> Task callouts).